index.html

```diff
@@ -30,49 +31,53 @@ <h1 class="foo-bar"><a href="#">Some Student</a></h1>
     <div class="z__container">
       <h2>
         This is my portfolio!
-      h2>
+      </h2>
       <h2>
-        There may be many like it, so I should replace this text to show that I am reading.
+        I am in fact reading.
       </h2>
       <h2>
-        <a href="mailto:my-email@goes-here.com" class="is-this-a-link">Contact Me?
+        <a href="mailto:dwtimp@gmail.com" class="btn ">Contact Me?</a>
       </h2>
     </div>
   </div>
 </header>
 
-<section class="y y--a">
+<section class="section__container">
   <div class="y__container">
-    <h2 class="title title--jumbo">I'm a student at the Iron Yard</h2>
+    <h2 class="section--info">I'm a student at the Iron Yard</h2>
 
-    <p class="title title--normal">I should write something about myself here and make it crafty.</p>
+    <p class="title title--normal">I go to a lot of concerts.  Seriously.  If I were to start counting I'd say 250
+      would be a good starting point.</p>
   </div>
 </section>
 
-<section class="y>
+<section class="section__container">
   <div class="y__container">
     <h3 class="y__header">My First Day</h3>
 
     <hr class="line">
 
-    <p>Day one is hard. It should be tough and challenging. I'll write something about it here. Two paragraphs would be nice.</p>
+    <p>Day one is hard. It should be tough and challenging. Getting used to the new surroundings, new people, new material
+      is a little overwhelming. Started to feel more comfortable with things very quickly though.</p>
   </div>
 </section>
 
-<section class="y y--b">
+<section class="section__container">
   <div class="y__container">
-    <h2 class="title title--jumbo">This is my tagline</h2>
+    <h2 class="title title--jumbo">Tired, but wiser for the time</h2>
```

Comment:
Look for classes that might replace `title` and it should replace all instances of `title`